### PR TITLE
feat: oryx-search small screens behaviour

### PR DESCRIPTION
**Pull request by Sinucid**
On small screens input becomes floated and it's visibility controls by icon-button

closes to [HRZ-2734](https://spryker.atlassian.net/browse/HRZ-2734)

[HRZ-2734]: https://spryker.atlassian.net/browse/HRZ-2734?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/base/ui/form/select/src/stories/select-demo.stories.ts [hunk 1]
```diff
@@ -151,15 +151,15 @@ SelectDemo.argTypes = {
     table: { category: 'Layout' },
   },
   searchIconPosition: {
-    options: ['PREFIX', 'SUFFIX', 'NONE'],
+    options: ['prefix', 'suffix', 'none'],
```
Comment (tobi-or-not-tobi) [inline libs/base/ui/form/select/src/stories/select-demo.stories.ts]: we better use `SearchIconPosition` here

File: libs/base/ui/search/searchbox/src/searchbox.component.ts [hunk 2]
```diff
@@ -9,17 +9,19 @@ import { SearchboxController } from './searchbox.controller';
...
 
 export class SearchboxComponent
   extends LitElement
-  implements SearchOptions, FormControlOptions, AffixOptions
+  implements SearchAttributes, FormControlOptions, AffixOptions
 {
   static override styles = baseStyles;
 
+  @property({ reflect: true, type: Boolean }) open?: boolean;
+  @property({ attribute: 'not-floated' }) notFloated?: boolean;
```
Comment (tobi-or-not-tobi) [inline libs/base/ui/search/searchbox/src/searchbox.component.ts]: what about `float = true`?

Comment (tobi-or-not-tobi) [reply libs/base/ui/search/searchbox/src/searchbox.component.ts]: also, let's add `type: Boolean`

**Review by tobi-or-not-tobi — CHANGES_REQUESTED** [review]
Nice work. The floating search box is not visible due to a clash with css specificity since the not-floating selector was added.

File: libs/base/ui/search/searchbox/src/searchbox.component.ts (hunk 2 above)
Comment (Sinucid) [reply libs/base/ui/search/searchbox/src/searchbox.component.ts]: `xs-floated` and turned off by default

**Review by tobi-or-not-tobi — APPROVED** [review]

**Pull request merged**